server/api/shiftView.js

```diff
@@ -0,0 +1,178 @@
+const dbClient = require('./dbClient')
+const { verifyJWTAuthToken } = require('./dataHelper')
+
+const shiftViewGETCallback = async (req, res) => {
+  try {
+    await verifyJWTAuthToken(req, res)
+
+    const shiftViewData = req.body
+
+    if (!shiftViewData.date || !shiftViewData.positionId) {
+      throw new Error('Missing/Invalid', { cause: { error: { status: 400, message: 'Missing Required Data' } } })
+    }
+
+    // slices off the time
+    shiftViewData.date = shiftViewData.date.slice(0, 10)
+
+    const timeBlocks = await getTimeBlocks(shiftViewData)
+    const employees = await getEmployees(shiftViewData)
+
+    const formattedEmployees = []
+
+    for (let i = 0; i < employees.length; i++) {
+      const employeeInfo = {
+        employeeId: employees[i].userId,
+        employeeName: employees[i].userName
+      }
+
+      // push to the array
+      formattedEmployees.push(employeeInfo)
+    }
+
+    // sort employees alphabetically by first name
+    formattedEmployees.sort((a, b) => {
+      if (a.employeeName < b.employeeName) return -1
+      if (a.employeeName > b.employeeName) return 1
+      return 0
+    })
+
+    const formattedTimeBlocks = []
+
+    for (let i = 0; i < timeBlocks.length; i++) {
+      const timeBlockInfo = {
+        timeID: timeBlocks[i].timeId,
+        startTime: timeBlocks[i].timeStart,
+        endTime: timeBlocks[i].timeEnd,
+        positionName: timeBlocks[i].positionName,
+        assignedToID: timeBlocks[i].userId,
+        assignedToName: timeBlocks[i].userName
+      }
```

Comment:
Same idea this loop can be changed to a for of loop
i.e. for (const timeBlock of timeBlocks)